graphql/id_test.go

```diff
@@ -0,0 +1,36 @@
+package graphql
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMarshalID(t *testing.T) {
+	tests := []struct {
+		Name        string
+		Input       interface{}
+		Expected    string
+		ShouldError bool
+	}{
+		{
+			Name:        "int64",
+			Input:       int64(12),
+			Expected:    "12",
+			ShouldError: false,
+		},
```

Comment:
https://golang.org/pkg/math/#pkg-constants
how about `math.MaxInt64`?

Reply:
Added math.MaxInt64 and math.MinInt64 test cases